code/controllers/subsystem/traumas.dm

```diff
@@ -14,42 +14,43 @@ SUBSYSTEM_DEF(traumas)
 	//phobia types is to pull from randomly for brain traumas, e.g. conspiracies is for special assignment only
 	phobia_types = sortList(list("spiders", "space", "security", "clowns", "greytide", "lizards",
 						"skeletons", "snakes", "robots", "doctors", "authority", "the supernatural",
-						"aliens", "strangers", "birds", "falling", "anime", "insects"))
+						"aliens", "strangers", "birds", "falling", "anime", "insects", "ocky_icky"))
```

Comment:
As the comment above mentions, this is the pool for random traumas, so it's just a matter of omitting it from this list

Reply:
Wow... I'm really dumb and was doing some really dumb things to try to bypass that...
Thanks for point it out directly.